packages/x-charts/src/LineChart/LineChart.tsx

```diff
@@ -216,13 +230,25 @@ LineChart.propTypes = {
    * Color palette used to colorize multiple series.
    */
   colors: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.string), PropTypes.func]),
+  /**
+   * An array of object that can be used to populate series and axis data using there `dataKey` property.
+   */
   dataset: PropTypes.arrayOf(PropTypes.object),
   desc: PropTypes.string,
+  /**
+   * If true, the charts will not listen to the mouse move event.
+   * It might breaks interactive features, but save some computation power.
+   * @default false
+   */
```

Comment:
```suggestion
  /**
   * If `true`, the charts will not listen to the mouse move event.
   * It might break interactive features, but is improving performance.
   * @default false
   */
```